docs/disko-install.md

```diff
@@ -140,6 +140,8 @@ Add this to your flake.nix output:
 {
   nixosConfigurations.your-machine = inputs.nixpkgs.lib.nixosSystem {
     system = "x86_64-linux";
+    # to pass this flake into your configuration (see the example below)
+    specialArgs = {inherit self;};
     modules = [
       {
         # TODO: add your NixOS configuration here, don't forget your hardware-configuration.nix as well!
```

Comment:
Am I understand right we should add a reference to the [Example for a NixOS installer](https://github.com/nix-community/disko/blob/master/docs/disko-install.md#example-for-a-nixos-installer) module file AND user's NixOS configuration (which in it's turn should include hardware-configuration.nix) here?

If so, I could try to make such edits to express it more clearly (or you could do it yourself, as I'm completely new to Nix).